scripts/msct_register_landmarks.py

```diff
@@ -342,7 +342,9 @@ def getRigidTransformFromLandmarks(points_dest, points_src, constraints='Tx_Ty_T
     sct.printv('Center:\n' + str(points_src_barycenter))
     sct.printv('Translation:\n' + str(translation_array))
 
-    if verbose == 2:
+    if qc_path is not None:
```

Comment:
this changes the current behaviour of `sct_testing`: now these figures are generated whereas before they where not. This is strange because by default, `qc_path` should be `None`.

Reply:
So, no setting of `-qc` when calling subprocesses?

Reply:
i think it's good to pass the variable `qc_path` in general (we can discuss the case-by-case later), but in this case, its value should have been None (as defined in the parent call `sct_register_to_template`), so I don't get why it entered the `if` case

Reply:
Fixed:
- The function parameter `path_qc` had an invalid default value, which explains what you saw. 
- By default, qc is not passed to children (but this can be done by using an environment variable...)
- Note: msct_register_landmarks still doesn't use the QC module, it still does its own stuff (for now)